french/script/story/st_00.rpy

```diff
@@ -6,901 +6,901 @@ translate french s00_last_time_on_jpde_265b952a:
+    adt sun angry "Le White-{w=0.15} non, le {b}Red Fang{/b} va effacer l'ardoise...{w=0.5} Et alors, nous reconstruirons."
 
 # game/script/story/st_00.rpy:137
 translate french s00_birth_of_the_red_fang_7752d420:
 
     # rf_1 sun "To victory, sir!"
-    rf_1 sun ""
+    rf_1 sun "Pour la victoire, monsieur !"
 
 # game/script/story/st_00.rpy:138
 translate french s00_birth_of_the_red_fang_60f8acde:
 
     # rf_2 sun "For the Fang!{w=0.3} And Fauni everywhere!"
-    rf_2 sun ""
+    rf_2 sun "Pour le Fang !{w=0.3} Et aux fauni !"
 
 # game/script/story/st_00.rpy:140
 translate french s00_birth_of_the_red_fang_81ed75f8:
 
     # rf_3 sun "Glory to the {b}Butcher of Beacon{/b}!"
-    rf_3 sun ""
+    rf_3 sun "Gloire au {b}Boucher de Beacon{/b} !"
 
 # game/script/story/st_00.rpy:145
 translate french s00_birth_of_the_red_fang_abc1b8a6:
 
     # sk sun frown ".{w=0.3}.{w=0.3}.{w=0.5}cleansed of...{w=0.5} The rot."
-    sk sun frown ""
+    sk sun frown ".{w=0.3}.{w=0.3}.{w=0.5}Purifié de...{w=0.5} Cette pourriture."
 
 # game/script/story/st_00.rpy:148
 translate french s00_birth_of_the_red_fang_e6836acc:
 
     # sk sun neutral "I've heard enough."
-    sk sun neutral ""
+    sk sun neutral "J'en ai assez entendu."
 
 # game/script/story/st_00.rpy:159
 translate french s00_birth_of_the_red_fang_e04e71dc:
 
     # adt_i sun neutral "(Three in front of us by the stairs, four behind us by the entrance...)"
-    adt_i sun neutral ""
+    adt_i sun neutral "(Trois devant nous près des escaliers, quatre derrière nous près de l'entrée...)"
 
 # game/script/story/st_00.rpy:160
 translate french s00_birth_of_the_red_fang_2eee9db3:
 
     # rf_2 sun "We're surrounded...?!"
-    rf_2 sun ""
+    rf_2 sun "Nous sommes cernés... ?!"
 
 # game/script/story/st_00.rpy:161
 translate french s00_birth_of_the_red_fang_cf6ec5f9:
 
     # sk sun neutral "Lay down your weapons.{w=0.3} You and your men."
-    sk sun neutral ""
+    sk sun neutral "Baissez vos armes.{w=0.3} Toi et tes hommes"
 
 # game/script/story/st_00.rpy:162
 translate french s00_birth_of_the_red_fang_7b389797:
 
     # adt sun neutral "Betrayal, then."
-    adt sun neutral ""
+    adt sun neutral "Une trahison, donc."
 
 # game/script/story/st_00.rpy:163
 translate french s00_birth_of_the_red_fang_02ee7053:
 
     # sk sun neutral "I have no wish to kill you and turn you into a martyr, Adam...{w=0.5} But you've had more than enough warnings and I still lead the Fang.{w=0.3} Popularity or not."
-    sk sun neutral ""
+    sk sun neutral "Je ne souhaite aucunement te tuer et te transfirler en un martyr, Adam...{w=0.5} Mais tu as été averti bien plus de fois que nécessaire et je reste maîtresse du Fang.{w=0.3} Populaire ou non."
 
 # game/script/story/st_00.rpy:167
 translate french s00_birth_of_the_red_fang_e9d189c3:
 
     # sk nulla "There's no betrayal in reeling in the chain of a rabid dog."
-    sk nulla ""
+    sk nulla "Il n'y a aucune trahison à secouer la chaîne d'un chien enragé."
```

Comment:
"reeling in" https://dictionary.cambridge.org/dictionary/english/reel-in-out
"briser la chaîne..."?